FeedSX/screens/createPost/index.tsx

```diff
@@ -416,13 +535,82 @@ const CreatePost = (props: IProps) => {
           placeholderTextColor="#0F1E3D66"
           inputTextStyle={styles.textInputView}
           multilineField
+          inputRef={myRef}
           inputText={postContentText}
-          onType={val => {
-            setPostContentText(val);
-          }}
+          onType={handleInputChange}
           autoFocus={postToEdit ? true : false}
+          partTypes={[
+            {
+              trigger: '@', // Should be a single character like '@' or '#'
+              textStyle: {
+                color: '#007AFF',
+              }, // The mention style in the input
+            },
+          ]}
         />
 
+        {/* users tagging list */}
+        {allTags && isUserTagging ? (
+          <View
+            style={[
+              styles.taggingListView,
+              {
+                height: userTaggingListHeight,
+              },
+            ]}>
+            <FlashList
+              data={[...allTags]}
+              renderItem={({item}: any) => {
```

Comment:
rename `item` to `user`

Reply:
item is used as standard name in flashlist